api/apps/api/src/modules/access-control/projects-acl/project-acl.controller.ts

```diff
@@ -0,0 +1,56 @@
+import { apiGlobalPrefixes } from '@marxan-api/api.config';
+import {
+  Controller,
+  Get,
+  Req,
+  Param,
+  ParseUUIDPipe,
+  UseGuards,
+  Delete,
+  Patch,
+  Body,
+} from '@nestjs/common';
+import { JwtAuthGuard } from '@marxan-api/guards/jwt-auth.guard';
+import { ProjectAclService } from './project-acl.service';
+import { RequestWithAuthenticatedUser } from '@marxan-api/app.controller';
+import { ApiBearerAuth, ApiTags } from '@nestjs/swagger';
+import { UserRoleInProjectDto } from './dto/user-role-project.dto';
+
+@UseGuards(JwtAuthGuard)
+@ApiBearerAuth()
+@ApiTags('Projects-Users Roles')
+@Controller(`${apiGlobalPrefixes.v1}/roles/projects`)
+export class ProjectAclController {
+  constructor(private readonly projectAclService: ProjectAclService) {}
+
+  @Get(':projectId/users')
+  async findUsersInProject(
+    @Param('projectId', ParseUUIDPipe) projectId: string,
+    @Req() req: RequestWithAuthenticatedUser,
+  ): Promise<UserRoleInProjectDto[]> {
+    await this.projectAclService.checkUserIsOwner(req.user.id, projectId);
+
+    return await this.projectAclService.findUsersInProject(projectId);
+  }
+
+  @Patch('/users')
```

Comment:
Any reason why we're not using `projectId` as part of the route here? It would be more consistent with the other endpoints in this controller :)

Reply:
It's mainly because the DTO before didn't include the `projectId` and Kamil asked why it shouldn't. I included it on the `DTO` and figured it was not necessary in the `param` anymore (as it was not fetched from there) and the `updateUserInProject` method in the service was using the dto itself. But do you think for consistency I should add it back?

Reply:
Yes, keeping those alike seems a good reason.

I may overlooked that DTO does not have it from `service` perspective - as it needs it to complete the check. 

So - in controller layer, `projectId` may remain in path, while the data passed down to service layer should include it.